ansible-ipi-install/roles/node-prep/tasks/main.yml

```diff
@@ -1,6 +1,13 @@
 ---
 - include_tasks: 10_validation.yml
   tags: validation
+  
+- include_tasks: 15_validation_disconnected_registry.yml
+  delegate_to: localhost
```

Comment:
this doesn't seem right. I don't think we want to delegate anything to the host running the playbook. This should delegate_to if no option given to the provisioner host.

Reply:
removed.
pushed in latest commit